zebra-rpc/src/methods/get_block_template_rpcs.rs

```diff
@@ -315,6 +322,10 @@ where
             let miner_fee = miner_fee(&mempool_txs);
 
             let block_height = (tip_height + 1).expect("tip is far below Height::MAX");
+            let block_time = tip_time
+                .checked_add_signed(Duration::seconds(POST_BLOSSOM_POW_TARGET_SPACING))
+                .expect("tip plus a small constant is far below i64::MAX")
+                .timestamp();
```

Comment:
The target spacing is a goal, not a consensus rule.

The actual `min_time` consensus rule is:
> For each block other than the genesis block , nTime MUST be strictly greater than the median-time-past of that block.

https://zips.z.cash/protocol/protocol.pdf#blockheader

This time will need to come from the state request.